testsuite/openshift/metrics/prometheus.py

```diff
@@ -70,24 +72,38 @@ def get_metrics(self, key: str = "", labels: dict[str, str] = None) -> Metrics:
 
         return Metrics(response.json()["data"]["result"])
 
-    def wait_for_scrape(self, target_service: str, metrics_path: str = "/metrics"):
+    @backoff.on_predicate(backoff.constant, interval=10, jitter=None, max_tries=35)
```

Comment:
35 seems too much. Works with 20 on my machine. Should we start with a lower number and gradually increase it based on the results? 

Or maybe you already tested it and it is the best choice :))

Reply:
Well, 20 works on my machine as well, but it keeps steadily failing in nightlies. The number is completely made up and aim is to fix the failing tests as we know it is just instability of ServiceMonitors

Reply:
fair, 35 is fine. We can tweak it later